### ISSUE TITLE
Unicode

### ISSUE THREAD

**Issue by nu8** [h3rald/min#64]
It seems multi-byte characters are not currently allowed. Here is working
example with single byte character:

~~~
[D:/Desktop/min_v0.20.1_windows_x64]$ "~"
 "~"
~~~

and fail example with `Ç (U+00C7)` (Alt + 128):

~~~
[D:/Desktop/min_v0.20.1_windows_x64]$ "Ç"
(!) [prompt]: <repl> [l:1, c:1] ERROR - Undefined value: ''
~~~

**Comment by h3rald** [comment]
Yep, that's a current limitation of nimline, the ReadLine alternative I created specifically for min.

The (half) good news is that unicode characters are handled (almost) correctly in min source files. Almost because the following output 2, not 1 as you might expect (it counts the bytes, not the characters).

What I should do is try including the [unicode](https://nim-lang.org/docs/unicode.html) nim module and try to see if I can improve things.
